discord/channel_update.go

```diff
@@ -94,17 +95,27 @@ func (GuildStageVoiceChannelUpdate) channelUpdate()      {}
 func (GuildStageVoiceChannelUpdate) guildChannelUpdate() {}
 
 type GuildForumChannelUpdate struct {
-	Name                 *string                `json:"name,omitempty"`
-	Position             *int                   `json:"position,omitempty"`
-	Topic                *string                `json:"topic,omitempty"`
-	PermissionOverwrites *[]PermissionOverwrite `json:"permission_overwrites,omitempty"`
-	ParentID             *snowflake.ID          `json:"parent_id,omitempty"`
-	RateLimitPerUser     *int                   `json:"rate_limit_per_user,omitempty"`
+	Name                          *string                              `json:"name,omitempty"`
+	Position                      *int                                 `json:"position,omitempty"`
+	Topic                         *string                              `json:"topic,omitempty"`
+	NSFW                          *bool                                `json:"nsfw,omitempty"`
+	PermissionOverwrites          *[]PermissionOverwrite               `json:"permission_overwrites,omitempty"`
+	ParentID                      *snowflake.ID                        `json:"parent_id,omitempty"`
+	RateLimitPerUser              *int                                 `json:"rate_limit_per_user"`
+	AvailableTags                 *[]ForumTag                          `json:"available_tags,omitempty"`
+	DefaultReactionEmoji          *json.Nullable[DefaultReactionEmoji] `json:"default_reaction_emoji,omitempty"`
+	DefaultThreadRateLimitPerUser *int                                 `json:"default_thread_rate_limit_per_user,omitempty"`
 }
 
 func (GuildForumChannelUpdate) channelUpdate()      {}
 func (GuildForumChannelUpdate) guildChannelUpdate() {}
 
+type GuildForumThreadChannelUpdate struct {
+	GuildThreadUpdate
```

Comment:
I think we should just add the fields instead of embedding as it's annoying when creating a new struct